var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -10,7 +10,7 @@
 from spack.package import *
 
 
-class Gromacs(CMakePackage, CudaPackage):
+class Gromacs(CMakePackage, CudaPackage, ROCmPackage):
```

Comment:
We don't have `LANGUAGE HIP` in our code or use `hipcc`, so `ROCmPackage` seems to be unnecessary.